libraries/Filter/HarmonicNotchFilter.h

```diff
@@ -84,7 +84,8 @@ class HarmonicNotchFilterParams : public NotchFilterParams {
     enum class Options {
         DoubleNotch = 1<<0,
         DynamicHarmonic = 1<<1,
-        LoopRateUpdate = 2<<1,
```

Comment:
Eek - this is horrendous! So LoopRateUpdate == DynamicHarmonic at the moment, that's a pretty bad bug!

Reply:
actually no, we just miss a slot

Reply:
one is value 2, the other value 4. What is the problem?